freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/PasswordService.java

```diff
@@ -91,21 +91,20 @@ private void asyncSetPasswords(String operationId, String accountId, String acto
                 requests.add(triggerSetPassword(stack, stack.getEnvironmentCrn(), userId, password));
             }
 
-            List<SuccessDetails> success = new ArrayList<>();
-            List<FailureDetails> failure = new ArrayList<>();
             for (SetPasswordRequest request : requests) {
                 try {
                     waitSetPassword(request);
-                    success.add(new SuccessDetails(request.getEnvironment()));
+                    syncOperationStatusService.updateOperation(operationId, new SuccessDetails(request.getEnvironment()), null);
```

Comment:
I would prefer two separate methods to addSuccess and addFailure rather than passing in null. I think that would make behavior of the methods more clear.